packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -148,21 +146,26 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
         uint256 _withdrawalLiveness,
         address _collateralAddress,
         address _finderAddress,
+        address _fpFinderAddress,
         bytes32 _priceIdentifier,
         string memory _syntheticName,
         string memory _syntheticSymbol,
         address _tokenFactoryAddress,
         FixedPoint.Unsigned memory _minSponsorTokens,
         address _timerAddress,
         address _excessTokenBeneficiary
-    ) public FeePayer(_collateralAddress, _finderAddress, _timerAddress) nonReentrant() {
+    )
+        public
+        FeePayer(_collateralAddress, _finderAddress, _timerAddress)
+        FundingRateApplier(FixedPoint.fromUnscaledUint(1), _fpFinderAddress, _priceIdentifier, _timerAddress)
```

Comment:
for now, the initial funding rate is set to 1e18. should this be parameterized or do we always want it to start at 1?

Reply:
I think this should be parameterized, for example we might want to start the first rate such that longs pay shorts by default

Reply:
I think if we want to parameterize it then we should set the initial funding rate within the funding rate store. I've refactored the implementation of the `FundingRateApplier` to no longer take in any initial funding rate number; rather it calls `_getLatestFundingRate` within the constructor. If we want to seed a different value we should first set it in the funding rate store.

Reply:
>  If we want to seed a different value we should first set it in the funding rate store.
agreed